src/scripts/main.js

```diff
@@ -1,3 +1,42 @@
 'use strict';
 
-// write code here
+const list = document.querySelectorAll('li');
+
+function sortList(element) {
+  const elementArray = Array.from(element);
+
+  elementArray.sort((a, b) => {
+    let x = b.getAttribute('data-salary').substring(1);
+
+    x = Number(x.replace(',', ''));
```

Comment:
Try extracting helper function so your code is more readable:
```js
function salaryToNumber(string) {
  return +string.slice(1).replaceAll(',', '');
}
```

And you could use it here like:
```js
const employees = getEmployees(list);

employees.sort((a, b) => salaryToNumber(b.salary) - salaryToNumber(a.salary));
```